Buildings/Experimental/DHC/CentralPlants/Cooling/Controls/ChillerStage.mo

```diff
@@ -84,11 +84,12 @@ model ChillerStage
     final t=1)
     "On signal of chiller two"
     annotation (Placement(transformation(extent={{120,-20},{140,0}})));
-  Buildings.Controls.OBC.CDL.Continuous.LessThreshold thrOneToTwo(
+  Buildings.Controls.OBC.CDL.Continuous.GreaterThreshold
```

Comment:
Please run modelica-fmt for this package to avoid unnecessary format changes to the models.